test/run_tests.sh

```diff
@@ -42,8 +42,12 @@ function run_opbyop {
 }
 
 function run_dynamic {
-  echo "Running in DynamicShape mode: $@"
-  XLA_EXPERIMENTAL="nonzero:masked_select" run_test "$@"
+  if [[ "$TPUVM_MODE" == "1" ]]; then
+    run_test "$@"
```

Comment:
Nit: should we log we're not running in dynamic mode in this case?

Reply:
My thinking is that we didn't log any thing for 
```
function run_test {
  "$@"
}
```
so if nothing is logged, we can assume it is running in the default mode